examples/manifests/consumer.yaml

```diff
@@ -49,8 +49,12 @@ spec:
           env:
             - name: HW_PLUGIN
```

Comment:
consumer should not have any Plugin valraibles

Reply:
only producer uses plugins

Reply:
updated